src/service/token/TokenService.ts

```diff
@@ -53,39 +55,46 @@ export class TokenService {
 	}
 
 	isLPToken(token: string) {
-		return [TOKENS.UNI_BADGER, TOKENS.UNI_DIGG, TOKENS.SUSHI_BADGER, TOKENS.SUSHI_DIGG, TOKENS.SUSHI_WBTC].includes(
-			token,
-		);
+		return [
+			TOKENS.UNI_BADGER_WBTC,
+			TOKENS.UNI_DIGG_WBTC,
+			TOKENS.SUSHI_BADGER_WBTC,
+			TOKENS.SUSHI_DIGG_WBTC,
+			TOKENS.SUSHI_ETH_WBTC,
+		].includes(token);
 	}
 
 	// TODO: More flexibly look up pools (sushi / uni share subgraph schema)
-	async getLiquidtyPoolTokenBalances(poolAddress: string, protocol: string, prices: any): Promise<TokenBalance[]> {
+	async getLiquidtyPoolTokenBalances(sett: SettData, settSnapshot: SettSnapshot): Promise<TokenBalance[]> {
+		const { depositToken, protocol } = sett;
+
 		let poolData;
 		if (protocol === 'uniswap') {
-			poolData = await getUniswapPair(poolAddress);
+			poolData = await getUniswapPair(depositToken);
 		}
 		if (protocol === 'sushiswap') {
-			poolData = await getSushiswapPair(poolAddress);
+			poolData = await getSushiswapPair(depositToken);
 		}
 		if (!poolData || !poolData.data) {
-			throw new NotFound(`${protocol} pool ${poolAddress} does not exist`);
+			throw new NotFound(`${protocol} pool ${depositToken} does not exist`);
 		}
 		const pair = poolData.data.pair;
+		const valueScalar = (settSnapshot.supply * settSnapshot.ratio) / pair.totalSupply;
 		const token0: TokenBalance = {
 			name: pair.token0.name,
 			address: pair.token0.id,
 			symbol: pair.token0.symbol,
 			decimals: pair.token0.decimals,
 			balance: pair.reserve0,
-			value: getUsdValue(pair.token0.id, pair.reserve0, prices),
+			value: (await this.priceService.getUsdValue(pair.token0.id, pair.reserve0)) * valueScalar,
```

Comment:
This was showing the full pool liquidity, so we needed to scale it down to the portion the sett owns.

Reply:
Maybe document this in the code for future reference?